cmd/dep/status.go

```diff
@@ -152,6 +158,29 @@ func (out *tableOutput) BasicLine(bs *BasicStatus) error {
 	return err
 }
 
+func (out *tableOutput) DetailHeader(metadata *dep.SolveMeta) error {
+	_, err := fmt.Fprintf(out.w, "PROJECT\tSOURCE\tCONSTRAINT\tVERSION\tREVISION\tLATEST\tPKGS USED\n")
+	return err
+}
+
+func (out *tableOutput) DetailFooter(metadata *dep.SolveMeta) error {
+	return out.BasicFooter()
+}
+
+func (out *tableOutput) DetailLine(ds *DetailStatus) error {
+	_, err := fmt.Fprintf(out.w,
```

Comment:
Hi @zjs,

Are you familiar with [`text/tabwriter`](https://golang.org/pkg/text/tabwriter/)?

Reply:
I was not. That seems like a better approach, but this pattern is copied from `func (out *tableOutput) BasicLine`. I think keepings things consistent is probably safest, and I think moving to using `text/tabwriter` probably belongs in a separate PR (or, at least, a separate commit).